training/distributed_training/pytorch/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>Are you able to programmatically fetch the account ID given the region, instead of hard-coding?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>We actually need to log in to this account to get the DLC.</p>

Reply:
<p>If you want to stick with this specific account ID, please also provide which region you used for testing this notebook and remind readers that they might have to change the account id depending on the <a href="https://github.com/aws/deep-learning-containers/blob/master/available_images.md#available-deep-learning-containers-images" target="_blank">AWS Region</a>.</p><p> </p>

Reply:
<p>Based on feedback from SMDDP team, the account <code>763104351884</code> is the one that hosts the publicly released DLC required for SMDDP. Then, we should not suggest to change it. It is ok for customers to use a different region and that is why we provide it as a parameter.</p>

Reply:
Depending on AWS Regions, the account ID has to be changed in some cases. Please have a look at the link: https://github.com/aws/deep-learning-containers/blob/master/available_images.md#available-deep-learning-containers-images

You don't need to add a code to automatically detect the ID right now, but it would be great if you can add a note like the following:

**Note:** The account ID of the PyTorch DLC image might vary depending on AWS Regions. To look up the right DLC image URI and account ID for your Region, see [Available Deep Learning Containers Images](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#available-deep-learning-containers-images).

Reply:
<p>Thank you for the feedback. I have added the comment and modified the impacted files in case the account ID is changed by the customer.</p><p>I structured in a way that the customer can focus on it and make a clear decision about the DLC account ID.</p><p> </p><p>I also agree that the documentation in the official docs should be clarified to mention that the account ID can be different based on the region.</p>